data/data/bootstrap/bootstrap-in-place/files/usr/local/bin/install-to-disk.sh.template

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env bash
+set -euoE pipefail ## -E option will cause functions to inherit trap
+
+# This script is executed by install-to-disk service when installing single node with bootstrap in place
+
+echo "Waiting for /opt/openshift/.bootkube.done"
+until [ -f /opt/openshift/.bootkube.done ]; do
+  sleep 5
+done
+
+if [ ! -f coreos-installer.done ]; then
+  # Write image + ignition to disk
+  echo "Executing coreos-installer with the following options: install -i /opt/openshift/master.ign {{.BootstrapInPlace.InstallationDisk}}"
+  coreos-installer install -i /opt/openshift/master.ign {{.BootstrapInPlace.InstallationDisk}}
++
+  touch coreos-installer.done
+fi
+
+if [ ! -f reboot.done ]; then
+  echo "Going to reboot"
+  shutdown -r +1 "Bootstrap completed, server is going to reboot."
+  touch reboot.done
```

Comment:
Does this `reboot.done` survive the reboot?  If not, do we want to drop it and the `if` guard here, so if for some reason the shutdown sticks and the unit/script gets restarted, it will call `shutdown` again?

Reply:
It will not survive the reboot.
I added this just to mark that the script executed the reboot command.
I agree there is little value to it and I'll just remove it.